app/serializers/users.js

```diff
@@ -4,3 +4,5 @@ exports.signUp = data => ({
   last_name: data.lastName,
   email: data.email
 });
+
+exports.usersList = data => data.map(user => exports.signUp(user));
```

Comment:
Suggestion, looking at this it does not make so much sense that it is a serializer since you are using the signUp serializer in the method, for which I consider that it is preferable that you move the map to the controller, and have after consulting the list of users something like this

const usersResponse = users.map (user => serializersignUp (user));